packages/browser/core/src/core/auth/internal/StitchAuthImpl.ts

```diff
@@ -433,7 +431,7 @@ export default class StitchAuthImpl extends CoreStitchAuth<StitchUser>
   private parseRedirect(): ParsedRedirectFragment | never {
     if (typeof this.jsdomWindow === "undefined") {
       // This means we're running in some environment other
-      // than a browser - so handling a redirect makes no sense here.
+      // Than a browser - so handling a redirect makes no sense here.
```

Comment:
[aside] IMO this is a dumb rule because it makes the comments more confusing if they span multiple lines and they're not multiline comments.

I would be in favor of disabling this rule globally but let's have a discussion with the whole team.